pandas/tests/frame/test_constructors.py

```diff
@@ -2698,6 +2698,14 @@ def test_frame_from_dict_with_mixed_tzaware_indexes(self):
 
 
 class TestDataFrameConstructorWithDtypeCoercion:
+    def test_none_values_on_string_columns(self):
+        # Creating DataFrame with column "a" as string and a None value
```

Comment:
These comments are not needed. Instead could you add a comment with the github issue number?